attic/rfc_reduction.md

```diff
@@ -0,0 +1,116 @@
+
+```
+ Feature Name: `Forecasting Reduction Techniques`
+ Start Date: 2019-11-16
+ Target Date:
+ ```
+
+## Summary
+[summary]: #summary
+ 
+ - Lagged values of a timeseries can be used as features for a machine learning algorithm.
```

Comment:
Can you expand more on what this RFC is about in particular? I'm not sure I have a clear understanding